man/cc-test-reporter-format-coverage.1.md

```diff
@@ -33,8 +33,7 @@ tests were run. Defaults to current working directory.
 
 ## COVERAGE_FILE
 
-Path where the coverage file is located. It requires the flag `--input-type`
-to be set.
+(Optional) Explicit path where the coverage file is located. If not passed, the path to the file is inferred based on standard some coverage tool locations. If specifying an explicit COVERAGE_FILE, an `--input-type` must also be specified.
```

Comment:
> i think this is valid usage, and it will search for the harcoded lcov file path and the simplecov file

I thought about that and initially dismissed it because I thought it would be a rare use case. Nonetheless, if it's usage is straightforward without a passed coverage file, I'm fine with that too.